Yoast/javascript
# Improve Czech stopwords list

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Fixes the stopwords list for Czech by adding the correct stopwords. Adds punctuation marks as sentence breakers.
* [Yoast SEO Free] Improves the Czech passive voice assessment by splitting sentences based on punctuation marks.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

Run `yarn test` and make sure that everything pass.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=99&modal=detail&selectedIssue=LINGO-801
